#// Структура для углов наклонов
int32 status # статус датчика
SEuler angleEuler 	# Углы Эллера